java/test/jmri/jmrit/logix/SCWarrantTest.java

```diff
@@ -104,12 +104,20 @@ public void testWarrant() throws JmriException {
         jmri.util.ThreadingUtil.runOnLayout(() -> {
             try {
                 sWest.setState(Sensor.INACTIVE);
+            } catch (jmri.JmriException e) {
+                Assert.fail("Unexpected Exception: " + e);
+            }
+        });
+        jmri.util.JUnitUtil.releaseThread(this, 100); // What should we specifically waitFor?
```

Comment:
I will merge this PR, but if you want to remove `JUnitUtil.releaseThread()`, which would be good, you can use `JUnitUtil.waitFor(int time)`.